modules/web/src/app/node-data/basic/provider/kubevirt/component.ts

```diff
@@ -632,4 +653,34 @@ export class KubeVirtBasicNodeDataComponent
       } as NodeSpec,
     } as NodeData;
   }
+
+  private _getQuotaCalculationPayload(instanceTypeId?: string): ResourceQuotaCalculationPayload {
+    const payload = {
+      replicas: this._nodeDataService.nodeData.count,
+      kubevirtNodeSize: {
+        [Controls.PrimaryDiskSize]: this.form.get(Controls.PrimaryDiskSize).value + 'G',
+      } as KubeVirtNodeSize,
+    };
+
+    if (instanceTypeId) {
```

Comment:
**Notes for Reviewer:**

- CPU, Memory should be string therefore `+ ' '` is added to convert into string.
Secondly:
- With Instance type -> "Gi" is passed based on instance type spec
- With Manual input. -> need to pass `M` so that backend converts can utilize `M` and return correct value in GB
- PrimaryDiskSize -> should be in GB + string value therefore `G` is passed